src/staging/gatsby-node.js

```diff
@@ -104,6 +104,7 @@ exports.createSchemaCustomization = ({ actions }) => {
     parkFacilities: [StrapiParkFacilities]
     parkOperation: StrapiParkOperation
     parkOperationSubAreas: [StrapiParkOperationSubAreas]
+    seo: StrapiSeoComponent
   }
```

Comment:
I think you'd need to define type for `seo` at `site` collection. Adding `seo: StrapiSeoComponent` to `type StrapiSites implements Node` should work.

Reply:
As above, for sites we're just using the component to populate the title and the description based on existing fields. So I think it's ok to not register it here. I might be wrong though, so if you know better, let me know.